include/zephyr/drivers/pm_cpu_ops.h

```diff
@@ -52,19 +52,6 @@ int pm_cpu_off(void);
  */
 int pm_cpu_on(unsigned long cpuid, uintptr_t entry_point);
 
-/**
- * @brief Power down the system
- *
- * This call is used to power down the whole system.
- *
- * A compliant PSCI implementation will never return, but some real-world
- * implementations do return errors in some cases.
- *
- * @retval does not return on success, a negative errno otherwise
- * @retval -ENOTSUP If the operation is not supported
- */
-int pm_system_off(void);
```

Comment:
I think `arch/arm64/core/fatal.c` is still using `pm_system_off`.

Reply:
fixed

Reply:
what is gonna happen with the rest of this API ? What is the plan ? Will we have `sys_shutdown()` and `pm_system_reset()` ? And the others API here ?